os/se/sss/security_hal_wrapper.c

```diff
@@ -51,7 +51,12 @@
 
 #define ISP_CHECKBUSY() while (isp_get_status()) {}
 
-#define HWRAP_LOG printf
+#ifdef CONFIG_HAL_WRAPPER_DEBUG
+#define HWRAP_LOG(format, ...) printf(format, ##__VA_ARGS__)
```

Comment:
Please don't use `printf` in kernel. Please use xxdbg like other kernel module.

Reply:
Thanks for the comment. I will update soon.